decomp/syntax/dependency.py

```diff
@@ -73,16 +73,11 @@ def from_conll(cls,
         depgraph.add_nodes_from([cls._conll_node_attrs(treeid, row, spec)
                                  for row in conll])
 
-        # add root node attributes
-        id_word = {nodeattr['position']-1: nodeattr['form']
-                   for nodeid, nodeattr in depgraph.nodes.items()}
-        sentence = [id_word[i] for i in range(max(list(id_word.keys()))+1)]
-
+        # add the root
         depgraph.add_node(treeid+'root-0',
                           position=0,
                           domain='root',
-                          type='root',
-                          sentence=' '.join(sentence))
```

Comment:
The deletion of this line doesn't break the sentence attribute of `UDSGraph` (or what is presumably now an attribute of `UDSSentenceGraph`)?

Reply:
No, because I made it an attribute of the `UDSSentenceGraph` (see [here](https://github.com/decompositional-semantics-initiative/decomp/pull/11/files#diff-35aabbdd15a0a61ceab491d4e41ba1b2R1191)).

Reply:
I remember that discussion, but I believe we decided not to use f-strings just in case. I think everything else is technically 3.5 compatible. (I just know that, e.g., the JHU grid still uses 3.5, so my suspicion is that other shared systems might use similar versions and may not support virtual environments.)

Reply:
I remember that discussion, but I believe we decided not to use f-strings then. I think everything else is technically 3.5 compatible. (I just know that, e.g., the JHU grid still uses 3.5, so my suspicion is that other shared systems might use similar versions and may not support virtual environments.)